xla/service/gpu/tests/gpu_fused_mha_test.cc

```diff
@@ -91,7 +89,7 @@ class MultiHeadedAttentionTest : public GpuCodegenTest {
   DebugOptions GetDebugOptionsForTest() override {
     auto debug_options = HloTestBase::GetDebugOptionsForTest();
     debug_options.set_xla_gpu_enable_xla_runtime_executable(false);
-    debug_options.set_xla_gpu_enable_cudnn_fmha(false);
+    debug_options.set_xla_gpu_enable_cudnn_fmha(true);
```

Comment:
Nit: The code below would be more concise and easier to read if you just pass the `bool config_with_fmha` into this function.

Reply:
This function is an override, it does not accept parameters. Functions like ParseAndReturnVerifiedModule apply debug options set by GetDebugOptionsForTest automatically. You are right though, code can be simplified further, I did that.